spaces/S000171/properties/P000018.md

```diff
@@ -0,0 +1,9 @@
+---
+space: S000171
+property: P000018
+value: true
+rrefs:
```

Comment:
Sorry, but there are four more rrefs.

Reply:
Should have guessed this was a copypaste error on all the properties. Thanks.